src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -296,15 +295,67 @@
                       "property": "description",
                       "dataType": { "dataType": "stringType" },
                       "queryable": true,
-                      "valueGetter": "( SELECT array_agg(elems.value ->> 'description') FROM jsonb_array_elements(org.jsonb->'urls') AS elems)",
                       "filterValueGetter": "( SELECT array_agg(lower(elems.value ->> 'description')) FROM jsonb_array_elements(org.jsonb -> 'urls') AS elems)",
                       "valueFunction": "lower(:value)"
+                    },
+                    {
+                      "name": "category_ids",
+                      "property": "categories",
+                      "dataType": {
+                        "dataType": "arrayType",
+                        "itemDataType": { "dataType": "rangedUUIDType" }
+                      },
+                      "queryable": false,
+                      "filterValueGetter": "( SELECT array_agg(lower(elems#>>'{}')) FROM jsonb_array_elements(org.jsonb->'urls') AS url_obj, jsonb_array_elements(url_obj->'categories') as elems )"
+                    },
+                    {
+                      "name": "category_names",
+                      "property": "categoriesNames",
+                      "dataType": {
+                        "dataType": "arrayType",
+                        "itemDataType": { "dataType": "stringType" }
+                      },
+                      "queryable": true,
+                      "filterValueGetter": "( SELECT array_agg(lower(cats.jsonb->>'value')) FROM jsonb_array_elements(org.jsonb->'urls') as url, jsonb_array_elements(url->'categories') as category_id JOIN src_organization_categories cats ON cats.id = (category_id#>>'{}')::uuid )"
                     }
                   ]
                 }
               },
-              "valueGetter": "org.jsonb ->> 'urls'",
+              "valueGetter": "( SELECT array_agg(url || jsonb_build_object('categoryNames', ( SELECT array_agg(cats.jsonb->>'value') FROM jsonb_array_elements(url->'categories') as category_id JOIN src_organization_categories cats on cats.id = (category_id#>>'{}')::uuid ))) FROM jsonb_array_elements(org.jsonb->'urls') as url)",
```

Comment:
This also does a little magic. Category names are not present in the original `jsonb` (only IDs), so we need to add them in.  In essence, we unroll the normal `urls` and apply similar logic as the `filterValueGetter` for the category names.

```sql

  SELECT array_agg(
      url || jsonb_build_object( -- || will merge two json objects, allowing us to add a categoryNames property to each
        'categoryNames',
        (
          SELECT array_agg(cats.jsonb->>'value')
          FROM jsonb_array_elements(url->'categories') as category_id
            JOIN src_organization_categories cats on cats.id = (category_id#>>'{}')::uuid
        )
      )
    )
  FROM jsonb_array_elements(org.jsonb->'urls') as url
```

Reply:
Result: we get category names here, too!
<img width="819" alt="image" src="https://github.com/folio-org/mod-fqm-manager/assets/8005215/d59d5e31-bbc0-4276-8238-271f7e9bda00">

Reply:
oops 
<img width="827" alt="image" src="https://github.com/folio-org/mod-fqm-manager/assets/8005215/f79aec4b-658e-4ad7-8e9e-026fd9469f92">

Reply:
fixed nesting issue. top comment still is good tho